pkg/network/java/testdata/docker-compose.yml

```
@@ -0,0 +1,7 @@
+version: '3'
+services:
+  java:
+    image: ${IMAGE_VERSION}
+    entrypoint: java -cp /v ${ENTRYCLASS}
+    volumes:
+      - ${TESTDATA_DIR}:/v
```

volumes are relative to docker-compose locations
```suggestion
    volumes:
      - ./:/v
```

---

done